tests/limits.rs

```diff
@@ -41,6 +41,7 @@ fn check_multiple_connection_refused() {
         init_connection_handler: DefaultInitConnection {},
         optional_features: PeerNetFeatures::default(),
         message_handler: DefaultMessagesHandler {},
+        max_message_size_read: 1048576000,
```

Comment:
Could there be tests that have a very low `max_message_size_read` to check it limits well ?
This new feature doesn't seam tested now

Reply:
done in limits.rs